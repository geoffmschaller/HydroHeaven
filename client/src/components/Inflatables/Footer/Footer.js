import React from 'react';
import styles from './Footer.module.css';
import MAIN_LOGO from '../../../static/images/main_logo_desktop_shaddow.png';

const Footer = () => (
	<div className={styles.footer}>
		<img src={MAIN_LOGO} alt=""/>
		<div className={styles.address}>135 Traffic Way - Arroyo Grande - (805) 473 - 2205</div>
		<div className={styles.address}>200 Higuera Street - San Luis Obispo - (805) 544 - 1772</div>
		<div className={styles.copyright}>&copy; {new Date().getFullYear()} Hydro Heaven Spas. All rights reserved.</div>
	</div>
);

export default Footer;
