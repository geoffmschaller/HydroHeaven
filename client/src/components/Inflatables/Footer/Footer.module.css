.footer {
  width: 100%;
  padding: 50px 0;
  text-align: center;
  color: white;
  background: #1f618d;
  background: -moz-radial-gradient(circle, #1f618d 0%, #154360 100%);
  background: -webkit-radial-gradient(circle, #1f618d 0%, #154360 100%);
  background: -o-radial-gradient(circle, #1f618d 0%, #154360 100%);
  background: radial-gradient(circle, #1f618d 0%, #154360 100%);
  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr="#1f618d", endColorstr="#154360", GradientType=1); }
  .footer img {
    width: 400px;
    margin: 0 auto;
    margin-bottom: 20px; }
  .footer .address {
    font-size: 18px; }
  .footer .copyright {
    font-size: 10px; }

@media (max-width: 500px) {
  .footer {
    padding: 50px 10px; }
    .footer img {
      width: 100%; } }
