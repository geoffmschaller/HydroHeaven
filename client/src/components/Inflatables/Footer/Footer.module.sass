$text-color: white

.footer
	width: 100%
	padding: 50px 0
	text-align: center
	color: $text-color
	background: rgb(31, 97, 141)
	background: -moz-radial-gradient(circle, rgba(31, 97, 141, 1) 0%, rgba(21, 67, 96, 1) 100%)
	background: -webkit-radial-gradient(circle, rgba(31, 97, 141, 1) 0%, rgba(21, 67, 96, 1) 100%)
	background: radial-gradient(circle, rgba(31, 97, 141, 1) 0%, rgba(21, 67, 96, 1) 100%)
	filter: progid:DXImageTransform.Microsoft.gradient(startColorstr="#1f618d", endColorstr="#154360", GradientType=1)


	img
		width: 400px
		margin: 0 auto
		margin-bottom: 20px

	.address
		font-size: 18px

	.copyright
		font-size: 10px

@media(max-width: 500px)
	.footer
		padding: 50px 10px

		img
			width: 100%