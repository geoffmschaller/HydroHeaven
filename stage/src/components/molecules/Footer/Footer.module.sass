@import '../../../themes/Main'

.footer
	width: 100%
	margin-top: 100px
	text-align: center
	padding: 10px

.dark
	color: $primary