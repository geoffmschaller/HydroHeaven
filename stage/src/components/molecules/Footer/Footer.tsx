import React, {FunctionComponent} from 'react';
import styles from './Footer.module.sass';
import FadeAndSlideUp from "../../../animations/FadeAndSlideUp";

interface Props {
	isDark: boolean
}

const Footer: FunctionComponent<Props> = (props) => {
	return (
		<FadeAndSlideUp>
			<div className={[styles.footer, props.isDark ? styles.dark : ""].join(" ")}>
				&copy; Kaloosian INC. {new Date().getFullYear()}. All Rights
				Reserved.
			</div>
		</FadeAndSlideUp>

	)
}

export default Footer;