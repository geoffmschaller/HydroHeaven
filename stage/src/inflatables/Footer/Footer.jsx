import React from 'react';
import styles from './Footer.module.sass';

import MAIN_LOGO from '../../static/images/main_logo_desktop_shaddow.png';

const Footer = () => {

	return (
		<div className={styles.footer}>
			<img src={MAIN_LOGO} alt=""/>
			<div className={styles.socialBar}>
				<div className={[styles.social, styles.facebook].join(" ")}><i className="fab fa-facebook-f"/></div>
				<div className={[styles.social, styles.instagram].join(" ")}><i className="fab fa-instagram"/></div>
				<div className={[styles.social, styles.yelp].join(" ")}><i className="fab fa-yelp"/></div>
			</div>
			<div className={styles.locations}>
				<div className={styles.location}>135 Traffic Way, Arroyo Grande - (805) 473 - 2205</div>
				<div className={styles.location}>200 Higuera Street, Sand Luis Obispo - (805) 544 - 1772</div>
			</div>
			<div className={styles.copy}>&copy;Kaloosian INC - {new Date().getFullYear()}. All Rights Reserved.</div>
		</div>
	);

};

export default Footer;