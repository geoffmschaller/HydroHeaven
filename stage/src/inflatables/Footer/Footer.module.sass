@import "src/static/sass/config"

.footer
	width: 100%
	padding: 75px 10px
	margin-top: 100px
	background-color: $primary
	text-align: center

	img
		width: 100%
		max-width: 400px

	.socialBar
		width: 100%
		max-width: 300px
		margin: 20px auto 0 auto
		display: flex
		flex-direction: row
		justify-content: space-between

		.social
			width: 30%
			padding: 5px
			color: white
			font-size: 30px
			transform: skew(-45deg)
			cursor: pointer

			&:hover
				padding-left: 10px

			i
				transform: skew(45deg)

		.facebook
			background-color: $facebook

		.instagram
			background-color: $instagram

		.yelp
			background-color: $yelp

	.locations
		width: 100%
		margin-top: 30px
		color: white

	.copy
		font-size: 12px
		color: $description-color