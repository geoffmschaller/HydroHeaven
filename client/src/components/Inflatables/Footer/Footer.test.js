import React from 'react';
import Footer from "./Footer";

const wrapper = <Footer/>;

it("Footer Inflatable", () => {
	expect(wrapper).toMatchSnapshot();
});