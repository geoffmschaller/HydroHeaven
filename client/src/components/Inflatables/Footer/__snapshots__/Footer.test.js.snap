// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Inflatable 1`] = `<Footer />`;
